File: src/main/java/org/bmsoft/graphics/Screen.java
package org.bmsoft.graphics;

import java.util.Random;

public class Screen {
    private final int width;
    private final int height;

    public int[] pixels;

    public Screen(int width, int height) {
        this.width = width;
        this.height = height;
        pixels = new int[this.width * this.height];
    }

    public void render() {
        for (int y = 0; y < this.height; y++) {
            for (int x = 0; x < this.width; x++) {
                pixels[x + y * width] = new Random().nextInt();
            }
        }
    }

    public void clear() {
        for(int i = 0; i < pixels.length; i++) {
            pixels[i] = 0x000000;
        }
    }
}

File: src/main/kotlin/org/bmsoft/graphics/Screen.kt
package org.bmsoft.graphics

import java.util.*
import kotlin.random.Random

class Screen(private val width: Int, private val height: Int) {
    var pixels: IntArray = IntArray(this.width * this.height)

    fun render() {
        for (y in 0 until this.height) {
            for (x in 0 until this.width) {
                pixels[Random.nextInt(width) + Random.nextInt(height) * width] = 0xFFFFFF
            }
        }
    }

    fun clear() {
        for (i in pixels.indices) {
            pixels[i] = 0
        }
    }
}